css/_buttons.scss

```diff
@@ -16,3 +16,58 @@
 .user-avatar {
   outline: none;
 }
+.ratting-btn {
+  background-color: $prime-color;
+  color: $bkg-tcolor-2;
+  border-radius: 4px;
+  border: 1px solid $prime-color;
+  outline: none;
+  padding: 0;
+}
+.ratting-btn:hover {
+  background-color: $bkg-tcolor-2;
+  color: $prime-color;
+  border: 1px solid $prime-color;
+  outline: none;
+}
+.ratting-btn:focus {
+  background-color: $bkg-tcolor-2;
+  color: $prime-color;
+  border: 1px solid $prime-color;
+  outline: none;
+}
+.ratting-btn:active {
+  background-color: $second-color;
+  color: $bkg-tcolor-2;
+  border: 1px solid $bkg-tcolor-2;
+  outline: none;
+}
```

Comment:
Need optimization. Common properties should be moved to common css rule.
`.ratting-btn:hover,
 .ratting-btn:focus,
 .ratting-btn:active {
   // common properties here
 }`

Reply:
Do you mean the outline properties?

Reply:
I've customized focus effect of all buttons, so I decided to delete 'outline' in btns effects and added 'outline: none' to custom reset_scss file.